Framework/PythonInterface/test/python/plugins/algorithms/FindUBFromScatteringPlaneTest.py

```diff
@@ -0,0 +1,171 @@
+import unittest
+from mantid.simpleapi import (
+    SetUB,
+    CreatePeaksWorkspace,
+    FindUBFromScatteringPlane,
+    LoadEmptyInstrument,
+    AddPeakHKL,
+    ClearUB,
+    IndexPeaks,
+    AnalysisDataService,
+)
+import numpy as np
+from unittest.mock import patch
+
+
+class FindUBFromScatteringPlaneTest(unittest.TestCase):
+    def setUp(self):
+        # load instrument + empty peakworkspace table
+        self.ws = LoadEmptyInstrument(Filename="SXD_Definition.xml", OutputWorkspace="empty_SXD")
+        axis = self.ws.getAxis(0)
+        axis.setUnit("TOF")
+        self.peaks1 = CreatePeaksWorkspace(InstrumentWorkspace=self.ws, NumberOfPeaks=0, OutputWorkspace="peaks1")
+
+    def assert_vector(self, ref_vector, calculated_vector, tol):
+        self.assertTrue(np.allclose(ref_vector, calculated_vector, atol=tol))
+
+    def index_peaks_helper(self, PeaksWorkspace, tolerance):
+        nindexed, avg_hkl_er, *_ = IndexPeaks(PeaksWorkspace, tolerance, RoundHKLs=False)
```

Comment:
Looks like `avg_hkl_er` is not used?
```suggestion
        nindexed,  *_ = IndexPeaks(PeaksWorkspace, tolerance, RoundHKLs=True)
```